### ISSUE TITLE
Fix issues with `blitz new` output

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#3544]
### Problem 1

Legacy output:
```
/p/tmp> blitz new myapp
✔ Pick a new project's language · TypeScript
? Pick your new app template …
❯ Full - includes DB and auth (Recommended)
  Minimal — no DB, no auth
```

Current output:

```
/p/tmp> blitz new myapp
✔ Pick which language you'd like to use for your new blitz project › TypeScript
? Pick which template you'd like to use for your new blitz project › - Use arrow-keys. Return to submit.
❯   full
    minimal
```

Current output questions are too wordy. And the `full` and `minimal` options are missing the explanations we previously had

**Output wording should be:**
```
/p/tmp> blitz new myapp
✔ Pick a language for the project · TypeScript
? Pick your new app template …
❯ Full - includes DB and auth (Recommended)
  Minimal — no DB, no auth
```

### Problem 2

In the `blitz new` output as seen below, we need:

- [ ] Extra line **before** "Hang tight while we set up your new Blitz app!"
- [ ] Extra line **after** "Hang tight while we set up your new Blitz app!"

```
/p/tmp> blitz new brandon
✔ Pick which language you'd like to use for your new blitz project › TypeScript
✔ Pick which template you'd like to use for your new blitz project › full
✔ Install dependencies? › pnpm
✔ Pick which form you'd like to use for your new blitz project › React Final Form
Hang tight while we set up your new Blitz app!
CREATE    .env
CREATE    .env.local
CREATE    .eslintrc.js
CREATE    README.md
```

**Comment by tech-chieftain** [comment]
@flybayer would love to work on this. So assigned it to me, please.

**Comment by abuuzayr** [comment]
all yours @tech-chieftain ! let us know if you need any help.

**Comment by beerose** [comment]
Hey @tech-chieftain, did you have a chance to work on this issue?

**Comment by tech-chieftain** [comment]
@beerose life got in the way and I couldn't get to do it. Expect a PR over the weekend.

**Comment by beerose** [comment]
Ok, thanks for the update @tech-chieftain!
